go.mod

```diff
@@ -52,25 +51,32 @@ require (
 	github.com/mitchellh/cli v1.1.1
 	github.com/openshift-metal3/terraform-provider-ironic v0.2.6
 	github.com/openshift/api v0.0.0-20211119153416-313e51eab8c8
-	github.com/openshift/client-go v0.0.0-20210409155308-a8e62c60e930
+	github.com/openshift/client-go v0.0.0-20211025111749-96ca2abfc56c
 	github.com/openshift/cloud-credential-operator v0.0.0-20200316201045-d10080b52c9e
+	github.com/openshift/cluster-api-provider-alibaba v0.0.0-20211123142315-d16daa4e6349
 	github.com/openshift/cluster-api-provider-gcp v0.0.1-0.20201203141909-4dc702fd57a5
 	github.com/openshift/cluster-api-provider-ibmcloud v0.0.0-20211008100740-4d7907adbd6b
 	github.com/openshift/cluster-api-provider-libvirt v0.2.1-0.20191219173431-2336783d4603
-	github.com/openshift/cluster-api-provider-ovirt v0.1.1-0.20210817084941-2262c7c6cece
-	github.com/openshift/library-go v0.0.0-20210408164723-7a65fdb398e2
-	github.com/openshift/machine-api-operator v0.2.1-0.20210505133115-b7ef098180db
+	github.com/openshift/cluster-api-provider-ovirt v0.1.1-0.20211111151530-06177b773958
+	github.com/openshift/library-go v0.0.0-20210811133500-5e31383de2a7
 	github.com/openshift/machine-config-operator v0.0.0
 	github.com/ovirt/go-ovirt v0.0.0-20210308100159-ac0bcbc88d7c
 	github.com/ovirt/terraform-provider-ovirt v0.99.1-0.20211019085223-db1ac552ec57
 	github.com/pborman/uuid v1.2.0
 	github.com/pkg/errors v0.9.1
 	github.com/pkg/sftp v1.10.1
+<<<<<<< HEAD
```

Comment:
Unresolved conflicts.

Reply:
I noticed this moved to go 1.17.  Updated.

Thanks!